ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  cvp_instance_ip:
+    documentation_options:
+      filename: Management Settings
+      table: CloudVision Settings
+    type: str
+    description: |
```

Comment:
Maybe in this doc should we encourage to use the `cvp_instance_ips` ?

Reply:
Should we deprecate this key, then?

Reply:
I will move this to an issue and we can address it when we revisit the docs.